apps/search/style/search.css

```diff
@@ -297,6 +299,14 @@ html[dir="rtl"] #suggestions-provider {
   padding: 0 1rem 0 2rem;
 }
 
+html[dir="rtl"] #suggestions li {
+  background-position: right 0.8rem center;
+}
+
+html[dir="rtl"] .local .result:after {
+  right: 0.5rem;
```

Comment:
Why do we need this right offset in RTL mode, but not in LTR mode? Ideally we would not need this RTL special rule, and instead we could just specify both `right` and `left` as `.5rem` and allow layout to take care of which rule to apply.